create-branch.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+branch_name=$1
+
+# checks if branch name is provided
+if [ -z "$branch_name" ]; then
```

Comment:
@yeaitsurya, can you add some description in this PR to describe how to run the shell script